source/user-manual/ruleset/ruleset-xml-syntax/regex.rst

```diff
@@ -2,19 +2,33 @@
 .. meta::
   :description: Learn more about regular expressions or regex. The Wazuh Ruleset is used to analyze incoming events and generate alerts when appropriate. 
   
-.. _regex_syntax:
-
 Regular Expression Syntax
 =========================
 
-**Regular expressions** or ``regex`` are sequences of characters that define a pattern.
+Regular expressions, or regex, are sequences of characters that define a pattern. In the Wazuh ruleset, some of the options used in rules definitions take a regular expression as their value. For example:
+
+.. code-block:: xml
+
+   <match>^Installed|^Updated|^Erased</match>
+
+There are three types of regular expressions that are recognized within the Wazuh ruleset.
+
+-  OS_Regex or regex syntax
+-  OS_Match or sregex syntax
+-  PCRE2 syntax
+
+The type values ``osregex``, ``osmatch``, and ``pcre2`` can be set  using the ``type`` key. For example:
+
+.. code-block:: xml
+
+   <srcport type="pcre2">^5000[0-7]$</srcport>
 
-There are three types of regular expressions: regex (*OS_Regex*), sregex (*OS_Match*) and PCRE2.
+For most of the options using regular expressions, ``osmatch`` is the type used by default. Check the :doc:`/user-manual/ruleset/ruleset-xml-syntax/rules` for more information.

```

Comment:
Maybe we could add a comment or a note saying that all the characters that aren't named explicitly in the tables are considered just as they are without any special meaning. 

It's like a generalization of this note below

```
. matches a literal dot, whereas \. matches any character
```